fdkaac.rc

```diff
@@ -1,3 +1,37 @@
 #include <winresrc.h>
 
 CREATEPROCESS_MANIFEST_RESOURCE_ID RT_MANIFEST "fdkaac.manifest"
+
+#define VER_VERSION 1,0,2,0
+//The version parameter is two 32-bit integers, defined by four 16-bit integers.
+#define VER_VERSION_STR "1.0.2"
+#define VER_LEGALCOPYRIGHT_STR "(c) nu774 2013-2021"
```

Comment:
I'm not sure if you like this copyright notice, you can modify it or delete this line and line 26 if you don't like it.